src/pages/ProfilePage/CardForm/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { IconButton, Box } from '@mui/material';
+
+import EditIcon from 'src/assets/icons/edit-grey.svg';
+import maskCardNumber from 'src/common/maskCardNumber';
+import { useAppSelector } from 'src/redux/hooks';
+import { useGetCardInfoQuery } from 'src/redux/user/userService';
+import { selectUserId } from 'src/redux/user/userSlice';
+
+import CardEditForm from '../CardEditFom';
+import { TitleStyled } from '../PersonalInformationForm/styles';
+
+import { SmallTitle, CardNumber } from './styles';
+
+function CardForm() {
```

Comment:
no this should came from stripe, its not secure create own card form

Reply:
can I leave it like this for demo and then will delete or substitute by stripe ?